src/main/webapp/mentors.html

```diff
@@ -19,4 +19,5 @@ <h1 class="display-4">Browse Available <span class="emphasis">Mentors</span></h1
   <script src="https://unpkg.com/prop-types@15.6/prop-types.js"></script>
   <!-- Load React components -->
   <script type="module" src="./js-build/common/react/display_navbar.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/mailtoui@1.0.3/dist/mailtoui-min.js"></script>
```

Comment:
```suggestion
  <script type="module" src="./js-build/common/react/display_navbar.js"></script>
  
  <script src="https://cdn.jsdelivr.net/npm/mailtoui@1.0.3/dist/mailtoui-min.js"></script>
```
nit. just to keep it a bit separated from the react stuff above.